docs/ProductName.md

```diff
@@ -43,6 +43,7 @@ The `ProductName` is a block responsible for **displaying the product name** alo
 | `showProductReference` | `Boolean` | Show product reference | `false`| 
 | `showBrandName` | `Boolean` | Show brand name | `false`| 
 | `classes` | `CustomCSSClasses` | Used to override default CSS handles. To better understand how this prop works, we recommend reading about it [here](https://github.com/vtex-apps/css-handles#usecustomclasses). Note that this is only useful if you're using this block as a React component. | `undefined` |
+| `showProductLink` | `Boolean` | Show the product link associated with the its name. | `false`| 
```

Comment:
Hi @anto90fg !
To keep our component's APIs as flexible as possible, without constantly making them bigger, we tend to stay away from boolean props.

In this case, to implement the functionality you want, how about creating a prop called `displayMode` (or something along those lines), which would accept the values `plain-text` and `link-to-product-page`? This way, if we need to add support to a new variation of this in the future, we just need to make the same prop accept one more value, instead of creating a whole new prop.

I'm not sold on these names I;'ve suggested 😅  Maybe @icazevedo has a better idea for them

Reply:
I like the names, but I think they should be camel-cased. `plainText` and `linkToProductPage`.

Reply:
I fixed it, following the requested changes. You can find all changes in the "required changes" commit.

Reply:
Please update the docs to reflect these changes 🙏